shared-doc/build-and-run-the-quickstart-with-bootable-jar.adoc

```diff
@@ -17,6 +17,9 @@ You can use the Maven plug-in to build a {productName} bootable JAR, and then yo
                       <configuration>
                           <feature-pack-location>wildfly@maven(org.jboss.universe:community-universe)#${version.server}</feature-pack-location>
                           <layers>
+ifdef::custom-bootable-jar-layers[]
```

Comment:
please revert, no need to specify layers now (all snippets now use <layers>...</layers>)